### ISSUE TITLE
Expose extra command line options

### ISSUE THREAD

**Issue by seanm-fathomhq** [muno92/resharper_inspectcode#420]
Would it be possible to expose the `--no-build` and the `--caches-home` options? I am working in a large solution and trying to bring down the run time of the action with caching and it's a bit tricky at the moment because these options aren't exposed.

**Comment by muno92** [comment]
@seanm-fathomhq 

Add `--no-build` option and `--caches-home` option in version 1.9.0.
https://github.com/muno92/resharper_inspectcode/releases/tag/1.9.0

You can use it as follows.

Example)
```yaml
on: push

jobs:
  inspection:
    runs-on: ubuntu-latest
    name: Inspection
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Setup .NET
        uses: actions/setup-dotnet@v3
        with:
          dotnet-version: '7.0.x'
      - name: Restore
        run: dotnet restore
      - name: Inspect code
        uses: muno92/resharper_inspectcode@v1
        with:
          solutionPath: ./YourSolution.sln
          noCache: true
          cachesHome: /path/to/caches_home
```

**Comment by seanm-fathomhq** [comment]
Perfect, thank you 😀

**Comment by seanm-fathomhq** [comment]
Whoops, reopening this as I only just got to testing out the new options. It looks like the `noBuild` option is broken, returning the error `Specify either '--build' or '--no-build'`. Looking at your code, the base command includes the `--build` option, and later if `noBuild` is true it appends `--no-build` but doesn't remove `--build` so it clashes. Probably need to remove `--build` from the base command and add it in an `else` when checking `noBuild`

**Comment by muno92** [comment]
@seanm-fathomhq 
Oh, sorry!

I forgot `--build` option was already specified, and fixed it in version [1.9.1](https://github.com/muno92/resharper_inspectcode/releases/tag/1.9.1).

**Comment by seanm-fathomhq** [comment]
All good 😁 I should have tested it before closing to make sure. I have definitely tried it this time now though and it is working, thank you!
